apps/wekan/config.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "../schema.json",
+  "name": "Wekan",
+  "port": 8207,
```

Comment:
The port is used by some app. Use a random one like this `8678` which is not used instead.

Reply:
will use 8678